apps/web/src/components/Profile/components/LanguageProfile/Display.tsx

```diff
@@ -1,45 +1,11 @@
-import { MessageDescriptor, defineMessages, useIntl } from "react-intl";
+import { useIntl } from "react-intl";
 
 import { commonMessages } from "@gc-digital-talent/i18n";
-import { getOrThrowError } from "@gc-digital-talent/helpers";
 import { Ul } from "@gc-digital-talent/ui";
 import { FragmentType, getFragment, graphql } from "@gc-digital-talent/graphql";
 
-import { getEvaluatedLanguageLevels } from "~/utils/userUtils";
 import FieldDisplay from "~/components/FieldDisplay/FieldDisplay";
-
-import { getExamValidityOptions, getLabels } from "./utils";
-
-// A workaround is required to show the deprecated bilingual evaluation in the profile snapshot.
-// These changes will be removed a year from now. (Remove snapshot workaround for the bilingual evaluation field #9905)
```

Comment:
Does this PR change anything related to #9905 and the deleted comment?

Reply:
I'll have to see, the ticket is a little vauge and this one only deals with the frontend and not the backend (likle file downloads)